media-libs/libplacebo/libplacebo-0.6.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+SCM=""
+if [[ "${PV}" == "9999" ]]; then
+	SCM="git-r3"
+	EGIT_REPO_URI="https://github.com/haasn/libplacebo"
+else
+	KEYWORDS="~amd64"
+	SRC_URI="https://github.com/haasn/libplacebo/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+inherit ${SCM} meson ninja-utils multilib-minimal
+
+DESCRIPTION="Reusable library for GPU-accelerated image processing primitives"
+HOMEPAGE="https://github.com/haasn/libplacebo"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+IUSE="glslang +shaderc +vulkan"
+REQUIRED_USE="vulkan? ( || ( glslang shaderc ) )"
+
+RDEPEND="glslang? ( <dev-util/glslang-7.10[${MULTILIB_USEDEP}] )
+	shaderc? ( media-libs/shaderc[${MULTILIB_USEDEP}] )
+	vulkan? ( media-libs/vulkan-loader[${MULTILIB_USEDEP}] )"
+DEPEND="${RDEPEND}"
+
+DOCS=( README.md )
```

Comment:
I'm sorry if you know that already but you don't have to specify `DOCS` if all installed documentation happens to match the default (which includes `README*`).

Reply:
Okay, I'll remove it.